qbiff.c

```diff
@@ -1,13 +1,6 @@
 #include <sys/types.h>
 #include <sys/stat.h>
-#include <utmp.h>
-#ifndef UTMP_FILE
-#ifdef _PATH_UTMP
-#define UTMP_FILE _PATH_UTMP
-#else
-#define UTMP_FILE "/etc/utmp"
-#endif
-#endif
+#include "utmp.h"
```

Comment:
Please do not name this like the global file. While basically all compilers on Un*x systems handle local paths before globals ones when using "", they are not required to (and e.g. Visual Studio does not do this). Please rename it to something sensible and use that name here.